src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts

```diff
@@ -0,0 +1,75 @@
+import { LitElement, html, type TemplateResult, type CSSResultGroup, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import '@sbb-esta/lyne-elements/chip.js';
+import style from './describe-each-chip-list.scss?lit&inline';
+
+/**
+ * Captures two groups
+ * - simple => "key=value" patterns
+ * - complex => "key=( key=value-... )
+ */
+const paramsRegex = /(?<complex>[a-zA-Z]*=\(.*\))|(?<simple>[a-zA-Z]+=[a-zA-Z]*)/gm;
+
+type DescribeEachItem = {
+  key: string;
+  value: string;
+  input: string;
+  isBoolean: boolean;
+  isUndefined: boolean;
+  isComplex: boolean;
+};
+
+/**
+ *  Convert the `describeEach` test title format into a list of more readable chips.
+ *  Chips are displayed differently depending on the param they represent:
+ *  - If param is boolean, show a green/red chip with the param name;
+ *  - If param is undefined, show nothing;
+ *  - Else, show a default chip with "key = value";
+ */
+@customElement('app-describe-each-chip-list')
+export class DescribeEachChipList extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  @property()
+  public set testCaseName(name: string) {
+    this._chips = Array.from(name.matchAll(paramsRegex)).map(this._mapToChip);
+    this.requestUpdate();
```

Comment:
Is the requestUpdate necessary?

Reply:
I discovered that it's still necessary if you do not have a getter for the property :/

Reply:
We could just add the getter, but this is fine too.